docs/topics/auth/default.txt

```diff
@@ -961,6 +961,12 @@ implementation details see :ref:`using-the-views`.
     * ``extra_context``: A dictionary of context data that will be added to the
       default context data passed to the template.
 
+    * ``redirect_authenticated_user``: Boolean, if True authenticated users
+      accessing the login page will be redirected to
+      :setting:`settings.LOGIN_REDIRECT_URL <LOGIN_REDIRECT_URL>`.
+      If LOGIN_REDIRECT_URL redirect to a login page it will cause a loop and
```

Comment:
"If the redirect url redirects to the same login page, `ValueError` will be raised."